src/ActionModal/components/ActionModal.vue

```diff
@@ -618,7 +619,16 @@ export default {
       const { type, memo, ...properties } = this.transactionData
       await this.actionManager.setMessage(type, properties)
       try {
-        this.gasEstimate = await this.actionManager.simulate(memo)
+        if (!config.enableTxAPI) {
+          this.gasEstimate = await this.actionManager.simulate(memo)
+        } else {
+          this.gasEstimate = await this.actionManager.simulateTxAPI(
+            this.createContext(),
```

Comment:
what does this do @colw?

Reply:
The old functionality is still there, and you can opt-in to the transaction service by setting an environment variable when building the front end.

Reply:
he meant the context maybe. that is like a bunch of variables you need for the tx to be build

Reply:
yep, that's what i meant. but i don't see the function anywhere.

Reply:
It's around, line ~500. Not part of this PR though.